geomesa-utils/src/test/resources/log4j.xml

```diff
@@ -2,8 +2,8 @@
 <!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
 <log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/" debug="false">
     <appender name="CONSOLE" class="org.apache.log4j.ConsoleAppender">
-        <layout class="org.apache.log4j.PatternLayout">
-            <param name="ConversionPattern" value="[%d] %5p %c: %m%n"/>
+        <layout class="org.apache.log4j.EnhancedPatternLayout">
+            <param name="ConversionPattern" value="[%d] %5p %c{1}: %m%n"/>
```

Comment:
possibly - it's just for unit tests though so in general I think it would be ok. It makes the logs a lot shorter. We could also make it use just a single char per package like `o.l.g.a.d.AccumuloDataStore` if you'd prefer that.